packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -785,10 +787,32 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     // we don't have event for it, but want to keep the event handling interface, event is empty.
     _.invoke(this.props, 'onSelectedChange', {}, { ...this.props, value })
   }
+
+  /**
+   * if there is no value we use the placeholder value
+   * otherwise, for single selection we convert the value with itemToString
+   * and for multiple selection we return empty string, the values are rendered by renderSelectedItems
+   */
+  private getSelectedItemAsString = (value: ShorthandValue): string => {
```

Comment:
why do we need that parameter if we can get it from state as we did with the props? this method might as well be `getTriggerButtonMessage` or `getTriggerButtonText`. not important but was just curious why not like that. oh, and capital I in `If there is no value...`

Reply:
this method is also used here, but not merged yet:
https://github.com/stardust-ui/react/pull/839/files#r254290634

with a different parameter, so the proposed name would not apply anymore, open to other naming suggestions

Will add capital "I", thanks